runtime/gc_base/GCExtensions.cpp

```diff
@@ -249,7 +249,8 @@ MM_GCExtensions::computeDefaultMaxHeapForJava(bool enableOriginalJDK8HeapSizeCom
 {
 	OMRPORT_ACCESS_FROM_OMRVM(_omrVM);
 	uintptr_t maxMemoryValue = memoryMax;
-	if (OMR_CGROUP_SUBSYSTEM_MEMORY == omrsysinfo_cgroup_are_subsystems_enabled(OMR_CGROUP_SUBSYSTEM_MEMORY)) {
+	if ((OMR_CGROUP_SUBSYSTEM_MEMORY == omrsysinfo_cgroup_are_subsystems_enabled(OMR_CGROUP_SUBSYSTEM_MEMORY) && omrsysinfo_cgroup_is_memlimit_set()) 
+		|| testContainerMemLimit) {
 		if (omrsysinfo_cgroup_is_memlimit_set()) {
```

Comment:
does not seem like you need this second/original invocation of memlimit

Reply:
This line was removed in softmxTest (main repo), should be removed here as well.